metadata/society/state_agency_dispatch.md

```diff
@@ -0,0 +1,79 @@
+# Title
+
+Utah State Agency Dispatch
+
+<!--- thought: should the title be "Utah State Agency Dispatch Zones"? Wondering if people might think this is a points layer with the dispatch centers, rather than a polygon layer of the dispatch jurisdictions. -->
+
+## ID
+
+1690a8de-0135-435e-b6bf-43b972fed19f
+
+## Brief Summary
+
+Polygon dataset displaying areas of responsibility for the State Department of Public Communications Dispatch Centers.
+
+<!--- I've noticed the metadata always describes these zones as "areas of responsibility" rather than "jurisdictions" and I was curious if that was just the verbiage used, or if there is a specific reason for that. -->
+
+## Summary
+
+This dataset displays the areas of responsibility for the State Department of Public Communications Centers for dispatching state agencies, such as Utah Highway Patrol and the Utah Division of Wildlife Resources.
+
+## Description
+
+### What is the dataset?
+
+The Utah Department of Public Safety Communications Bureau serves as the dispatch partner for many state agencies, including Utah Highway Patrol, the Division of Wildlife Resources, and the Department of Corrections. Each polygon in this dataset represents the approximate geographic extent of each dispatch center's area of responsibility. It does not include the specific locations of law enforcement facilities or dispatch centers.
+
+### What is the purpose of the dataset?
+
+This dataset is primarily used in emergency services and public safety. Dispatch zones help determine which Public Communications dispatch center will respond to a 911 caller, per Next Generation 911 ([NG911](https://www.911.gov/issues/ng911/)) guidelines. You can learn more about how geospatial data are used in emergency response on our [blog](https://gis.utah.gov/blog/2021-04-09-ugrc-ng911-update/).
+
+### What does the dataset represent?
+
+Areas of responsibility for each Department of Public Communications Center in this dataset are represented as polygons and are labelled with the corresponding dispatch center.
+
+### How was the dataset created?
+
+This dataset was created based on information provided by the Communications Centers.
+
+<!--- Did we create this dataset? It seemed like we did, but I wasn't too sure. -->
+
+### How reliable and accurate is the dataset?
+
+This dataset is the most current and complete version of dispatch zones for state agencies in Utah. Though not a substitute for a legal land survey, this dataset is suitable for cartographic and general analytic purposes.
```

Comment:
Do we need to mention the stuff about legal land surveys? Also, is this used by the dispatch centers themselves, or is it just a general reference? If it is used by them, that would be a big positive for reliability.

Reply:
With the legal land survey disclaimer, I usually include it if it was mentioned in the original metadata, which it often is, but I suppose directing people to the general disclaimer should be sufficient. I don't think we necessarily need to include it in every layer.

As for your second question about whether the dispatch centers use these data, I'm not sure. @mhchou do you have any info on this?